whatiusedtodo.html

```diff
@@ -29,14 +34,14 @@ <h1 align='center'> What I Used to Do </h1>
 
 <div class='container'>
 	<div class='row'>
-		<div class='col-sm-6' style="text-align:center">
+		<div class='col-sm-6 col-xs-12' style="text-align:center">
```

Comment:
There's a bootstrap class called `text-center` you can use instead of inline styles. Avoid using them if you can, you'll end up having to change all of them if you ever want to make a change, and they override any CSS styles you apply